test/extended/cloud_controller_manager/ccm.go

```diff
@@ -0,0 +1,61 @@
+package cloud_controller_manager
+
+import (
+	"context"
+
+	"github.com/ghodss/yaml"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-apps][Feature:OpenShiftCloudControllerManager][Late]", func() {
+	defer g.GinkgoRecover()
+
+	g.It("TestCCMStatus", func() {
```

Comment:
This should be human readable as a sentence
```suggestion
	g.It("Deploy an external cloud provider", func() {
```